.github/workflows/image_build_and_test.yaml

```diff
@@ -121,20 +121,22 @@ jobs:
         env:
           XDG_DATA_HOME: /home/runner/work/tube
 
-      - name: Run ETL process
-        working-directory: ../compose-etl
-        run: docker-compose run tube bash -c "python run_config.py; python run_etl.py"
-
-      - name: Run integrated tests
+      - name: Run dataframe tests
         # Run the tests that require Spark and ElasticSearch
         env:
           ES_URL: localhost
           POSTGRES_PORT: 5432
-          DICTIONARY_URL: https://s3.amazonaws.com/dictionary-artifacts/ndhdictionary/3.3.8/schema.json
+          DICTIONARY_URL: https://s3.amazonaws.com/dictionary-artifacts/ibdgc-dictionary/1.6.10/schema.json
           XDG_DATA_HOME: /home/runner/work/tube
+          RUNNING_MODE: Test
+          HADOOP_VERSION: 3.3.2
           ES_INDEX_NAME: etl
+          ES_HADOOP_VERSION: 8.3.3
+          ES_HADOOP_HOME: /home/runner/work/tube
+          ES_SPARK_30_2_12: elasticsearch-spark-30_2.12
+          ES_SPARK_20_2_11: elasticsearch-spark-20_2.11
```

Comment:
why are there 2 versions?

Reply:
we have it here because spark can be either spark 2.0 or spark 3.0

Reply:
but which spark version are we using to run the tests?

and do you need to set  these environment variables here? i only see them referenced in the dockerfile

Reply:
let me recheck the gen3-spark docker, but I think we have it for some reason of compatibility when we move from spark 2 to spark 3

Reply:
checked it, we need both since we are using elasticsearch-spark-30_2.12 as elasticsearch-spark-20_2.11 for compatibility